string command
string typePlan
geometry_msgs/Pose goal
string goalJM
int64 grasp_width